doc/source/visualizing/colormaps/cmap_images.py

```diff
@@ -8,7 +8,7 @@
 # Create projections using each colormap available.
 p = yt.ProjectionPlot(ds, "z", "density", weight_field="density", width=0.4)
 
-for cmap in cm.datad:
```

Comment:
`matplotlib.cm.datad` isn't deprecated but it's also not clearly intended to be exposed, as it just happens to be there as a side effect of a re-export, see https://github.com/matplotlib/matplotlib/blob/9b1fcf67c4228c4a2788af5bcaf0c6fde09a55bf/lib/matplotlib/cm.py#L26

so I've replaced it with current API, which would break for old versions of Matplotlib, but I think it's okay to do that in a docs script ?